### PR TITLE
PReLU specification refactoring

### PR DESCRIPTION
**Pull request by ggalieroc**
PReLU specification refactored.

Details:
- Extended detailed description
- Modified the mathematical formula describing the op
- Added type of tensors
- Added example

Ticket: 45990

### CODE REVIEW

File: docs/ops/activation/PReLU_1.md [hunk 1]
```diff
@@ -2,32 +2,59 @@
...
+    x \quad \mbox{if } x \geq  0 \\
+    \alpha x \quad \mbox{if } x < 0
+\end{array}\right.
+\f]
 
-**Outputs**
+Where α, is a learnable parameter and corresponds to the negative slope defined by the second input `slope`.
 
-* **1**: The result of element-wise PReLU operation applied for tensor from input 1 with slope values from input 2. A tensor of type T1 and shape matching shape of input *x* tensor.
+Before the operation computation, the second input tensor `slope` is broadcasted to the first input tensor `data` based on [Broadcast Rules For Elementwise Operations](../broadcast_rules.md). Additionally, *PReLU* is equivalent to *ReLU* operation when `slope` is equal to zero input tensor.
```
Comment (lazarevevgeny) [inline docs/ops/activation/PReLU_1.md:24]: It is not true. The PReLU `slope` input is applied per-channel. So, the input tensor with shape `[1,32,46,46]` the slope shape will be `[32]`. So, we had a bug in the PReLU operation which you can fix now as part of the operation revise.

**Review by lazarevevgeny — CHANGES_REQUESTED** [review]

File: docs/ops/activation/PReLU_1.md (hunk 1 above)
Comment (ggalieroc) [reply docs/ops/activation/PReLU_1.md]: I have addressed the comment as requested. Please re-review

**Review by tsavina — CHANGES_REQUESTED** [review]

File: docs/ops/activation/PReLU_1.md [hunk 2]
```diff
@@ -2,32 +2,88 @@
...
+*PReLU* performs element-wise parametric *ReLU* operation on a given input tensor, based on the following mathematical formula:
+
+\f[
+PReLU(x) = \left\{\begin{array}{r}
+    x \quad \mbox{if } x \geq  0 \\
+    \alpha x \quad \mbox{if } x < 0
+\end{array}\right.
+\f]
+
+Where α, is a learnable parameter and corresponds to the negative slope, per channel, defined by the second input `slope`.
```
[the remark this answers is not in the document]
Comment (ggalieroc) [reply docs/ops/activation/PReLU_1.md]: Done

File: docs/ops/activation/PReLU_1.md [hunk 3]
```diff
@@ -2,32 +2,88 @@
...
+
+Another mathematical representation that may be found in other references is as follows:
```
[the remark this answers is not in the document]
Comment (ggalieroc) [reply docs/ops/activation/PReLU_1.md]: Agree, it sounded weird. Done

File: docs/ops/activation/PReLU_1.md [hunk 4]
```diff
@@ -2,32 +2,88 @@
...
 
-* **2**: `slope` - Tensor with negative slope values of type T2. The shape of the tensor should be broadcastable to input 1. Required.
+
+**Attributes**: *PReLU* operation has no attributes.
+
+**Inputs**
+
+* **1**: `data`. A tensor of type `T` and arbitrary shape. **Required**.
+* **2**: `slope`. 1D tensor of type `T`. Tensor with negative slope values, one per channel dimension of `data` input tensor. **Required**.
+* **Note**: Channels dimension corresponds to second dimension of `data` input tensor. If `data` rank is less than 2, then the number of channels is 1.
```
[the remark this answers is not in the document]
Comment (ggalieroc) [reply docs/ops/activation/PReLU_1.md]: Done

File: docs/ops/activation/PReLU_1.md [hunk 5]
```diff
@@ -2,32 +2,88 @@
...
 
 **Outputs**
 
-* **1**: The result of element-wise PReLU operation applied for tensor from input 1 with slope values from input 2. A tensor of type T1 and shape matching shape of input *x* tensor.
+* **1**: The result of element-wise *PReLU* operation applied to `data` input tensor with negative slope values from `slope` input tensor. A tensor of type `T` and same shape as `data` input tensor.
```
[the remark this answers is not in the document]
Comment (ggalieroc) [reply docs/ops/activation/PReLU_1.md]: Done

File: docs/ops/activation/PReLU_1.md [hunk 6]
```diff
@@ -2,32 +2,88 @@
...
 
 **Types**
 
-* *T1*: arbitrary supported floating point type.
+* *T*: arbitrary supported floating point type.
```
[the remark this answers is not in the document]
Comment (ggalieroc) [reply docs/ops/activation/PReLU_1.md]: Done

**Review by tsavina — APPROVED** [review]

File: docs/ops/activation/PReLU_1.md [hunk 7]
```diff
@@ -2,32 +2,88 @@
...
\ No newline at end of file
+```xml
+<layer ... type="Prelu">
+    <input>
+        <port id="0">
+            <dim>20</dim>
+            <dim>128</dim>
+        </port>
+        <port id="1">
+            <dim>20</dim>
```
Comment (lazarevevgeny) [inline docs/ops/activation/PReLU_1.md:79]: May be the shape should be 128?

File: docs/ops/activation/PReLU_1.md [hunk 8]
```diff
@@ -2,32 +2,88 @@
...
+        </port>
+    </input>
+    <output>
+        <port id="2">
+            <dim>20</dim>
+            <dim>128</dim>
+        </port>
+    </output>
+</layer>
+```
```
Comment (lazarevevgeny) [inline docs/ops/activation/PReLU_1.md:89]: Add an example with 4D case

File: docs/ops/activation/PReLU_1.md (hunk 7 above)
Comment (ggalieroc) [reply docs/ops/activation/PReLU_1.md:103]: Correct, my bad. Done

File: docs/ops/activation/PReLU_1.md (hunk 8 above)
Comment (ggalieroc) [reply docs/ops/activation/PReLU_1.md:115]: Done

**Review by lazarevevgeny — APPROVED** [review]

**Pull request merged**